src/Symfony/Component/Console/Helper/QuestionHelper.php

```diff
@@ -424,6 +420,32 @@ private function getShell()
     }
 
     /**
+     * Reads user input.
+     *
+     * @param resource $stream The input stream
+     *
+     * @return string User input
+     *
+     * @throws \RuntimeException
+     */
+    private function readFromInput($stream)
+    {
+        if (STDIN === $stream && function_exists('readline')) {
+            $ret = readline();
+        } else {
+            $ret = fgets($stream, 4096);
+
+            if (false === $ret) {
+                throw new \RuntimeException('Aborted');
+            }
+
+            $ret = trim($ret);
```

Comment:
By the way, the behaviour is now inconsistent as the string returned by `readline()` is not trimmed (for example, when containing extra spaces before or after the actual input).